recipes/uwebsockets/all/conanfile.py

```diff
@@ -49,17 +58,23 @@ def configure(self):
                 % (self.name, minimal_cpp_standard)
             )
 
+        # libdeflate is not supported before 19.0.0
+        if tools.Version(self.version) < "19.0.0":
+            self.options.with_libdeflate = False
```

Comment:
its better to delete option that dont make sense.

This way we avoid creating package ids that do not exist

Reply:
Thank you for review.
I agree with you.
I will fix it later.